packages/contracts/config/.env.example

```diff
@@ -1,27 +0,0 @@
-################
-# Instructions #
-################
-# Create environment file(s) named ".<environment>.env" in this same location using this file as a template
-# For instance `yarn run deploy:some-contract local` will use .local.env for configuration.
-
-
-############
-# Template #
-############
-
-### REQUIRED ###
-# Mnemonic for the wallet used to deploy the contracts
-DEPLOY_MNEMONIC='device frost invest worth exchange need fix accuse action pave head response'
-
-FORCE_INCLUSION_PERIOD=600
-
-# Private key for the Sequencer - note must have some ETH for gas fees
-SEQUENCER_PRIVATE_KEY='191c53d1b552bdd3f95a3129b1c5ec93bba1a92d756c1aa690dbe564d4b3ecf0'
-
-# Note: can use any network name. 'local' or leaving it blank will deploy to DEPLOY_LOCAL_URL
-DEPLOY_NETWORK='local'
-
-# Only if deploying locally
-DEPLOY_LOCAL_URL='http://127.0.0.1:8545'
```

Comment:
Did this config move to another place? Or is it just not required anymore?

Reply:
Ah I'm going to rewrite the deployment scripts since we've had some API changes in the contracts. So I'll include this as part of a new PR.